X/Xorg_xkeyboard_config/build_tarballs.jl

```diff
@@ -34,5 +35,11 @@ dependencies = [
     Dependency("Xorg_xkbcomp_jll"),
 ]
 
+init_block = raw"""
+if Sys.islinux() || Sys.isfreebsd()
+    ENV["XKB_CONFIG_ROOT"] = get(ENV, "XKB_CONFIG_ROOT", joinpath(artifact_dir, "share", "X11", "xkb"))
+end
+"""
+
 # Build the tarballs.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; init_block)
```

Comment:
AnyPlatform doesn't require changing Julia compat

Reply:
Ah, sorry, misread the line!

Reply:
Although there's actually a benefit in requiring Julia 1.6: we can safely drop Pkg from the dependencies of the package.

Reply:
Good to merge I think?

Reply:
ah, missed that last comment. Want me to add Julia 1.6 compat and remove the Pkg dep?

Reply:
Yeah, even if not strictly required, dropping Pkg from dependencies is a good thing